src/Formik.tsx

```diff
@@ -1,306 +1,240 @@
 import * as React from 'react';
 import isEqual from 'react-fast-compare';
-import warning from 'warning';
 import deepmerge from 'deepmerge';
-import { FormikProvider } from './connect';
 import {
-  FormikActions,
   FormikConfig,
   FormikErrors,
   FormikState,
   FormikTouched,
   FormikValues,
-  FormikContext,
   FormikProps,
 } from './types';
 import {
-  isEmptyChildren,
   isFunction,
-  isNaN,
-  isPromise,
   isString,
   setIn,
+  isEmptyChildren,
+  isPromise,
   setNestedObjectValues,
   getActiveElement,
   getIn,
   makeCancelable,
 } from './utils';
+import { FormikProvider } from './FormikContext';
+import warning from 'warning';
```

Comment:
Can this be replaced with tiny-warning? It has esm support.

Reply:
Fixed.